frameworks/README.md

```diff
@@ -0,0 +1,37 @@
+# URLs For Frameworks
+
+## Apfell
+
+- https://github.com/its-a-feature/Apfell
+
+## Covenant
+
+- https://github.com/cobbr/Covenant
+
+## Silver
+
+- https://github.com/BishopFox/sliver
+
+## FactionC2
+
+- https://github.com/FactionC2/Faction
+
+## Silent Trinity
+
+- https://github.com/byt3bl33d3r/SILENTTRINITY
+
+## PoshC2
+
+- https://github.com/nettitude/PoshC2_Python
+
+## Merlin
+
+- https://github.com/Ne0nd0g/merlin
+
+## Koadic
+
+- https://github.com/zerosum0x0/koadic
+
+## Awesome Red Teaming
+
+- https://github.com/yeyintminthuhtut/Awesome-Red-Teaming
```

Comment:
Please remove, this isn't a C2, it was in the urls file for automation purposes later down the road.